File: test1/src/main/java/com/example/test1/API/ApiServices.java
package com.example.test1.API;

import android.database.Observable;

import com.google.gson.JsonObject;

import retrofit2.http.POST;

/**
 * Created by Niken on 7/22/2020.
 */
public interface ApiServices {
    @POST(ApiUrl.Initiate)
        public Observable<JsonObject> initiate();
}

File: app/src/main/java/com/example/intentapp/API/ApiServices.kt
package com.example.intentapp.API

import com.google.gson.JsonObject
import io.reactivex.Observable
import retrofit2.http.Body
import retrofit2.http.GET
import retrofit2.http.POST

interface ApiServices {

    @GET(ApiUrl.getList)
    fun getList(): Observable<JsonObject>

    @POST(ApiUrl.Initiate)
    fun initiate(@Body req: JsonObject): Observable<JsonObject>
}